README.md

```diff
@@ -28,9 +28,8 @@ sudo apt install cmake bison
 You can use the standard iceoryx cmake build approach with an activated `-DDDS_STACK=CYCLONE_DDS`
 or `-DDDS_STACK=FAST_DDS` switch.
 ```sh
-cmake -Bbuild -DBUILD_TEST=ON -DDDS_STACK=CYCLONE_DDS # DDDS_STACK=FAST_DDS
-cd build
-make
+cmake -Bbuild -DBUILD_TEST=ON -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DDDS_STACK=CYCLONE_DDS # -DDDS_STACK=FAST_DDS
```

Comment:
@SchwarzeMagier I think the compile commands are not needed, right?

Reply:
It either needs to be set on command line or in the CMakeLists.txt. I would suggest to just add `set(CMAKE_EXPORT_COMPILE_COMMANDS ON)` at the top of the root CMakeLists.txt.